thingsboard.github.io/docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright © 2016-2020 The Thingsboard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '3.1'
+services:
+  thingsboard_github_io:
+    container_name: thingsboard_github_io
+    restart: always
+    image: "THINGSBOARD_GITHUB_IO_IMAGE"
```

Comment:
???

Reply:
Replaced by the latest thingsboard/website image.